### DISCUSSION TITLE
Dependancies

### DISCUSSION THREAD

**Discussion by gman77** [UI5/webcomponents#2874] [Q&A]
Hi,

I would like to know if there are any dependencies to using ui5-webcomponents. Could they be used in older frameworks (eg. Angular 5,6)? Or is there any browser dependency?

Thank you.

**Comment by fifoosid** [comment]
Hi @gman77 

There are no issues from ```ui5-webcomponents``` side. If the given framework supports the web standard [web components,](https://developer.mozilla.org/en-US/docs/Web/Web_Components) it means that they can be used. Nowadays, all frameworks support web components.

When it comes to browser support, ```ui5-webcomponents``` support all major browsers(Chrome, Firefox, Edge, Safari). However, it is worth mentioning that they no longer support IE11.
